sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/MapsSearchAsyncClient.java

```diff
@@ -100,6 +102,7 @@
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<List<Polygon>> getPolygons(List<String> geometryIds) {
         Mono<Response<List<Polygon>>> result = this.getPolygonsWithResponse(geometryIds);
+        Response<List<Polygon>> resp = result.block();
```

Comment:
This like does not belong here. Needs to be removed. It's best to reset this file to its original state.

Reply:
It's also blocking the flow of the client, so it cannot be here